src/uu/sort/src/sort.rs

```diff
@@ -1226,15 +1272,15 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
         settings.separator = Some(separator.chars().next().unwrap())
     }
 
-    if matches.is_present(OPT_KEY) {
-        for key in &matches.args[OPT_KEY].vals {
+    if matches.is_present(options::KEY) {
+        for key in &matches.args[options::KEY].vals {
```

Comment:
Is this another case like https://github.com/uutils/coreutils/issues/2304? You probably want something like `matches.values_of` right?